tests/unit/models/requests/test_ledger_entry.py

```diff
@@ -69,6 +69,30 @@ def test_has_only_ticket_is_valid(self):
         )
         self.assertTrue(req.is_valid())
 
+    def test_has_only_xchain_claim_id_is_valid(self):
+        req = LedgerEntry(
+            xchain_claim_id=1,
+        )
+        self.assertTrue(req.is_valid())
+
+    def test_has_only_xchain_create_account_claim_id_is_valid(self):
+        req = LedgerEntry(
+            xchain_create_account_claim_id=1,
+        )
+        self.assertTrue(req.is_valid())
+
+    def test_has_both_bridge_fields_is_valid(self):
+        req = LedgerEntry(
+            bridge=XChainBridge(
+                locking_chain_door="rGzx83BVoqTYbGn7tiVAnFw7cbxjin13jL",
+                locking_chain_issue=XRP(),
+                issuing_chain_door="r3kmLJN5D28dHuH8vZNUZpMC43pEHpaocV",
+                issuing_chain_issue=XRP(),
+            ),
+            bridge_account="rGzx83BVoqTYbGn7tiVAnFw7cbxjin13jL",
+        )
+        self.assertTrue(req.is_valid())
```

Comment:
Should there be any invalid test cases for `LedgerEntry` related to XChain features?